config/routes.rb

```diff
@@ -16,4 +16,6 @@
 
   draw :placements
   draw :claims
+
+  get "/healthcheck", to: proc { [200, {}, ["OK"]] }
```

Comment:
Maybe https://edgeapi.rubyonrails.org/classes/Rails/HealthController.html could be used?

```suggestion
  get "/healthcheck", to: "health#show"
```

Rails supposedly defaults this to `/up`, but locally it isn't in there for me.

Reply:
I've started reading that and figure out that we don't check if the DB is up or if we add sideqkiq and redis later on, we don't check the health of those. Refactored this into a `Heartbeat` controller, similar to the one in register. To check for these things